documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -4,6 +4,28 @@
 Release Notes
 #############
 
+6.3.11
+======
+* Added a hint field in the trace event when all replicas of some data are lost. `(PR #4209) <https://github.com/apple/foundationdb/pull/4209>`_
+* Rewrote SQLite injected fault handling. `(PR #4212) <https://github.com/apple/foundationdb/pull/4212>`_
+* Add a SevWarnAlways trace line to help debug a rare failure. `(PR #4214) <https://github.com/apple/foundationdb/pull/4214>`_
```

Comment:
These release notes seem to be written more as a development log rather than a list of changes relevant to people using the cluster. Having a log like this may be useful, but I suspect that it's also useful to have a version of the release notes that is more geared towards consumers like the ones we had before.

Reply:
` debug a rare failure` here is more like `diagnose a rare situation` that may happen in a cluster. 

Having it reported as SevWarnAlways that SevInfo can help SRE to identify the problem without the deep knowledge of how FDB (SS and DD) works. So IIHO, the PR should be mentioned in the release note. Maybe the wording should be better reflect what I described above to avoid confusion.